src/scripts/windows/azure-pipelines.yml

```diff
@@ -10,5 +10,15 @@ pool:
 container: build_container
 
 steps:
-- script: C:\BuildTools\Common7\Tools\VsDevCmd.bat && cmake ./src -B_build -GNinja && ninja -C _build -v
+- script: C:\BuildTools\Common7\Tools\VsDevCmd.bat && cmake ./src -B_build -GNinja -DCMAKE_INSTALL_PREFIX=install && ninja -C _build install -v
   displayName: 'build'
+- script: .\install\test\cpp_test.exe -r junit -o TEST-cpp_test.xml
+  displayName: 'cpp_test'
+  continueOnError: true
+- script: .\install\test\platform\test_platform.exe -r junit -o TEST-test_platform.xml
+  displayName: 'test_platform'
+  continueOnError: true
+- task: PublishTestResults@2
+  inputs:
+    testResultsFormat: 'JUnit'
+    testResultsFiles: '**/TEST-*.xml'
```

Comment:
Nit: newline

Reply:
I'm going to rework the azure pipelines files in the near future so we can have a single file drive both windows and ubuntu (and eventually macos) builds. Will fix then